lib/mandrill_dm/message.rb

```diff
@@ -248,6 +258,10 @@ def return_string_value(field)
       mail[field] ? mail[field].to_s : nil
     end
 
+    def return_time_parsed_string_value(field)
+      mail[field] ? Time.parse(mail[field].value).utc.strftime("%Y-%m-%d %H:%M:%S") : nil
```

Comment:
It is a bad idea to `Time.parse` an unknown input - we'll get a lot of parse errors. It's a user's responsibility to pass a proper string to Mandrill.
Just do the `.utc.strftime("%Y-%m-%d %H:%M:%S")` part if `value.is_a?(Time)` (`DateTime` and `Date` are also ok and should be converted to `Time`), otherwise pass it as it is.